utils/markdown.js

```diff
@@ -227,7 +227,7 @@ function render(nodes, maxColumns) {
  */
 function innerRenderMdNode(indent, node, lastNode, result, maxColumns) {
   const newLine = () => {
-    if (result.length && result[result.length - 1] !== '')
```

Comment:
Referring to this change @yury-s.

Reply:
I ran java api generator with api.json produced after this patch has been applied and it produced the same result as before (no diffs) so it is good to go.